kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -281,6 +298,6 @@ window.onhashchange = function () {
   };
 })(window.history);
 
-window.onpopstate = history.onpushstate = () => {
+window.onpopstate = () => {
```

Comment:
if `history.onpushstate` is removed does it still work if you switch between dashboard/discover/visualize?

Reply:
I have the same concern

Reply:
Yes, it has had no problems with switching in between so far. One thing I just found is that the `Reporting` menu does not show up in Discover until you select a Saved search though

Reply:
I think we need someone else to manually peer test this, as it's difficult to add related tests for in-context menu. This is the original PR that adds the code above https://github.com/opendistro-for-elasticsearch/kibana-reports/pull/172

Reply:
Sure- I posted my branch in the chat for peer-testing and have been manually testing myself as well

Reply:
Need some extra carefulness since we are about to cut the release

Reply:
This is an issue with `dev` branch as well, could you please first post PR there, and then backport to `7.9.1`?